extensions/xcode/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "xcode",
+  "title": "Xcode",
+  "version": "1.0.0",
+  "author": "Sven Tiigi",
```

Comment:
Here should be your Raycast account username. It should've been injected here if you used "Create Extension" command 🤔